webapp/package.json

```diff
@@ -18,7 +18,7 @@
     "marked": "mattermost/marked#cb85e5cc81bc7937dbb73c3c53d9532b1b97e3ca",
     "match-at": "0.1.0",
     "object-assign": "4.0.1",
-    "perfect-scrollbar": "0.6.10",
+    "perfect-scrollbar": "^0.6.10",
```

Comment:
Might want to remove the `^` so that we don't accidentally update perfectscrollbar when we don't want to